README.md

```diff
@@ -38,7 +38,7 @@ $ npm ci
 $ npm run build
 ```
 
-After that, the `dist` folder will contain all files required by the extension.
```

Comment:
Why did you change the name to `PrivacyPass`? I think the name `PrivacyPass` doesn't indicate that the files in the directory are generated files.

Reply:
because when Chrome is used to pack an extension, both the name of the .crx and the .pem are expected to be the same as the input directory (all siblings)..

`.gitignore` allows the .pem to be permanently homed where Chrome expects it to be.. without ever being commit to version control.. and likewise, the generated .crx and .xpi files won't either

this naming convention is associated with the scripts in the `.bin` directory.. if you already have tools that you use to package your extensions, then.. you:
1. won't want/need the `.bin` directory
   * though I find these scripts useful
2. won't want to rename the output directory.. as you tooling will already be configured to use the old path

Reply:
another option.. which might be preferable.. would be to:
* move the `.bin` directory to `dist/.bin`
* configure webpack to output to `dist/PrivacyPass`
* configure `.gitignore` to exclude: `dist/PrivacyPass` and `dist/PrivacyPass.[pem|crx|xpi]`

so all output.. and my scripts to pack the output into extensions.. would be confined under `dist`

Reply:
__update__:
That last suggestion seemed (to me) as a much cleaner option.. so I just pushed a commit to make those changes.